cmd/launcher/interactive.go

```diff
@@ -40,7 +41,7 @@ func runInteractive(args []string) error {
 	}
 
 	// have to keep tempdir name short so we don't exceed socket length
```

Comment:
Optional -- would we want to add a wrapper `agent.MkdirTempForSocket` (or whatever) that could hold a check for this?

Reply:
Don't see an issue with that, as long as folks are onboard with the wrapper idea in general

Reply:
🤷‍♀️ I don't feel strongly about it, might as well leave as is for now